templates/repo/actions/runs_list.tmpl

```diff
@@ -6,31 +6,35 @@
 	</div>
 	{{end}}
 	{{range .Runs}}
-		<li class="item gt-df gt-py-4">
+		<li class="item gt-df gt-py-3">
 			<div class="issue-item-left issue-item-icon gt-df gt-items-start">
 				{{template "repo/actions/status" (dict "status" .Status.String "locale" $.locale)}}
 			</div>
-			<div class="issue-item-main action-item-main gt-f1 gt-fc gt-df gt-mr-3">
+			<div class="issue-item-main action-item-main gt-f1 gt-fc gt-df gt-mr-3 gt-gap-2">
 				<div class="issue-item-top-row gt-mb-2">
 					<a class="index gt-no-underline title action-item-title" title="{{.Title}}" href="{{if .Link}}{{.Link}}{{else}}{{$.Link}}/{{.Index}}{{end}}">
 						{{- .Title -}}
 					</a>
 				</div>
 				<div class="desc issue-item-bottom-row gt-df gt-ac gt-fw">
-					<b>{{if not $.CurWorkflow}}{{.WorkflowID}} {{end}}#{{.Index}}</b>: {{$.locale.Tr "actions.runs.commit"}}
-					<a href="{{$.RepoLink}}/commit/{{.CommitSHA}}">{{ShortSha .CommitSHA}}</a>{{$.locale.Tr "actions.runs.pushed_by"}} {{.TriggerUser.GetDisplayName}}
-					<span class="ui label">
-						{{if .RefLink}}
-							<a href="{{.RefLink}}">{{.PrettyRef}}</a>
-						{{else}}
-							{{.PrettyRef}}
-						{{end}}
+					<b>{{if not $.CurWorkflow}}{{.WorkflowID}} {{end}}#{{.Index}}</b>
+					: {{$.locale.Tr "actions.runs.commit"}}
+					<a href="{{$.RepoLink}}/commit/{{.CommitSHA}}">{{ShortSha .CommitSHA}}</a>
+					{{$.locale.Tr "actions.runs.pushed_by"}}
+					<a href="{{.TriggerUser.HomeLink}}">{{.TriggerUser.GetDisplayName}}</a>
 					</span>
 				</div>
 			</div>
-			<div class="issue-item-right">
-				<div>{{TimeSinceUnix .Updated $.locale}}</div>
-				<div>{{.Duration}}</div>
+			<div class="action-item-center gt-df gt-ac gt-pr-4 gt-pl-2">
+				{{if .RefLink}}
+					<a class="ui label gt-px-2 gt-mx-0" href="{{.RefLink}}">{{.PrettyRef}}</a>
+				{{else}}
+					<span class="ui label gt-px-2 gt-mx-0">{{.PrettyRef}}</span>
+				{{end}}
```

Comment:
```suggestion				
				<a class="ui label gt-px-2 gt-mx-0" {{if .RefLink}}href="{{.RefLink}}"{{end}}>{{.PrettyRef}}</a>
```

The same as this? A link without "href" doesn't navigate.

Reply:
I'm concerned for accessibilty if we emit `a` without `href`.

Reply:
I moved a number of styles to shared ones in https://github.com/go-gitea/gitea/pull/25601/commits/54fe4d501c7649f9ac8bd8b98a133b1dd112fabf.

Reply:
Think I'll leave this as-is.